tests/ui_tests/test_add_new_contact_page.py

```diff
@@ -22,7 +21,7 @@
 
 
 @pytest.mark.add_new_contact_page
-@pytest.mark.usefixtures("del_all_contacts")
+@pytest.mark.usefixtures("del_all_contacts", "setup_user")
```

Comment:
There is only 1 class in this module

```suggestion
@pytest.mark.ui
@pytest.mark.add_new_contact_page
@pytest.mark.usefixtures("del_all_contacts", "setup_user")
```